2048-qt.pro

```diff
@@ -17,7 +17,7 @@ RESOURCES += \
     resources.qrc
 
 # Additional import path used to resolve QML modules in Qt Creator's code model
-QML_IMPORT_PATH =
+QML_IMPORT_PATH = /usr/lib/i386-linux-gnu/qt5/imports
```

Comment:
new locale should work without changing this

Reply:
Friend, it's been 6 years since I sent this PR, I don't believe anyone wants it to be merged